src/Runner/TestHandler.php

```diff
@@ -113,7 +113,7 @@ private function initiatePhpVersion($version, PhpInterpreter $interpreter)
 
 	private function initiatePhpIni($value, PhpInterpreter $interpreter)
 	{
-		$interpreter->arguments .= ' -d ' . Helpers::escapeArg($value);
+		$interpreter->addArguments(' -d ' . Helpers::escapeArg($value));
```

Comment:
Hmm. Now I cannot find it. I recall there were more options for format. Now manual talks about `=` only. Maybe they dopped it. Anyway, `addPhpIniOption('name=value')` could be fine too.